loompy/2.0.16/Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM python:3.9.1
+MAINTAINER Alex Sickler (sicklera@email.chop.edu)
```

Comment:
Please use `LABEL maintainer="Alex Sickler (sicklera@email.chop.edu)"`
As the `MAINTAINER` tag is deprecated

Reply:
Oh, also, in general instructions should be upper case - so `ADD` instead of `add`